modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/AbstractExtensionModelLoader.java

```diff
@@ -51,7 +51,7 @@ public abstract class AbstractExtensionModelLoader extends ExtensionModelLoader
   protected void configureContextBeforeDeclaration(ExtensionLoadingContext context) {
     context.addCustomValidators(validators);
 
-    if (IGNORE_DISABLED) {
+    if (IGNORE_DISABLED || context.isEnableIgnoredComponents()) {
```

Comment:
shouldn't the more specific context param override the system property?

Reply:
If the SysProp is set to true, but the context explicitly says `false`, the context should win.